kmk/extensions/peg_rgb_matrix.py

```diff
@@ -53,6 +56,9 @@ def __init__(
         self.disable_auto_write = disable_auto_write
         self.split = split
         self.rightSide = rightSide
+        self.brightness_step = 0.1
+        self.brightness = 0
```

Comment:
Add support for changing RGB LED brightness via code or through two new keycodes `RGB_BRI` and `RGB_BRD`.